src/operators/Clip/Clip.xml

```diff
@@ -74,7 +74,10 @@
       <Field name="radius" label="Radius" type="double" enabler="funcType:Sphere">
         1.000000
       </Field>
-      <Field name="sphereInverse" label="Inverse" type="bool" enabler="funcType:Sphere">
+      <Field name="sphereInverse" label="Inverse" type="bool" ignoreeq="true" enabler="funcType:Sphere">
```

Comment:
Can you clarify why we ignore the sphere inverse?

Reply:
Yes, why was this added?

Reply:
I actually have no idea. I'm pretty certain I didn't change this variable, but xmledit changed it for some reason when I re-generated the xml files. Is this a concern? I can look more into it when I'm back on VisIt if so.

Reply:
Yes, please investigate, If you didn't make the change purposefully, then it shouldn't have been made automagically either. Perhaps you accidentally clicked a checkbox ?

Reply:
Fixed!